src/constants.js

```diff
@@ -11,5 +11,17 @@ export const SUBJECT_TYPE = Object.freeze({
 
 export const EVENTS_BY_SUBJECT_TYPE = {
   [SUBJECT_TYPE.INPUT]: ['change'],
-  [SUBJECT_TYPE.DRAG_N_DROP]: ['dragstart', 'drag', 'dragenter', 'drop', 'change', 'dragleave', 'dragend'],
+  /**
+   * @see https://developer.mozilla.org/en-US/docs/Web/API/DragEvent
+   */
+  [SUBJECT_TYPE.DRAG_N_DROP]: [
```

Comment:
I'm fine with this change. The only think that IMO important is order – it is weird to have `dragend` triggered before `dragstart`

Reply:
Cool stuff, now all set!
I was checking `dragover` event specs [on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/dragover_event
) and have an impression that it is triggered **on the element that is dragged** into dropzone (or anywhere else), but not the dropzone itself.

https://jsfiddle.net/radonirinamaminiaina/zfnj5rv4/

Added console log there, trying to drop labeled div into any other down below:

<img width="926" alt="Screenshot 2021-04-08 at 6 02 24 PM" src="https://user-images.githubusercontent.com/11467765/114059363-93f7bb80-9894-11eb-86ae-2ff61fc2a5a4.png">

Can you please double check that?

Reply:
That sounds (and looks) correct. Should we remove `dragleave` as well? That was another event added, and it does not seem strictly applicable to the usage in this PR, so if we're removing `dragover` I think it makes sense to remove `dragleave` as well 🙂